extensions/binding/org.eclipse.smarthome.binding.onewire/ESH-INF/thing/DigitalIO.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onewire" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<thing-type id="digitalio">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="owserver" />
+		</supported-bridge-type-refs>
+		<label>Digital I/O</label>
+		<description>A single digital I/O device (DS2405)</description>
+		<channels>
+			<channel id="present" typeId="present" />
```

Comment:
Sometimes 1W busses tend to be instable (depending e.g. on cabling). If this happens the sensors seem to vanish and reappear later. This can be easliy monitored by this state. If it is possible to use persistence services with thing status, I can remove it.

Reply:
The thing status is always a live status. Why should it be persisted? Rules are already able to react on the thing status. On restart the thing status will again be set by each corresponding thing handler.